app/javascript/chat/channels.jsx

```diff
@@ -80,19 +80,23 @@ export const Channels = ({
   }
   let configFooter = '';
   if (expanded) {
-    // TODO: The <div /> below should be converted into a real menu or <nav />
     configFooter = (
-      <div className="chatchannels__config">
-        <img alt="configration" src={ConfigImage} style={{ height: '18px' }} />
-        <div className="chatchannels__configmenu" role="menu">
-          <a href="/settings" role="menuitem">
-            Settings
-          </a>
-          <a href="/report-abuse" role="menuitem">
-            Report Abuse
-          </a>
-        </div>
-      </div>
+      <nav className="chatchannels__config" aria-label="configuration menu">
+        <img
+          alt="configuration"
+          src={ConfigImage}
+          style={{ height: '18px' }}
+          aria-hidden="true"
+        />
```

Comment:
I'm not sure if there is a plan to cover the broader issues in another PR (just let me know if so!), but to make this component accessible, this should be contained in a button, and activate on click rather than hover.

That button should live outside of the `nav` element so that it can toggle the visibility of the whole nav rather than just the list of links (the result of not doing that is that we surface an empty navigation to screen reader users as per my other comment).

When a user opens the nav by keyboard, we should also be sending focus to the first link in the list / returning focus to the button when the menu closes. Marcy introduced a pattern for this in the top navigation (activated by profile image click) - it's in `app/javascript/topNavigation/utilities.js`

Like I say, happy if this comes under another issue, but just want to make sure it's all captured!

Reply:
Learning so much about accessibility by just reading this comment 😍

Reply:
Connect needs a lot of love in this respect. I'll see if I can tackle this in one PR, but if not, I'll definitely open an issue with your comments. Thanks!

Reply:
No worries - yeah there's a lot in there 😅
I guess in this case I can't decide if surfacing an empty nav is better than no nav at all, if you see what I mean 🙃  Totally get there's a lot to wade into in Connect though and if it makes sense to chop it into smaller pieces that's grand - all steps forward for sure!